bundles/net.tourbook/src/net/tourbook/ui/views/tourBook/ActionDeleteTour.java

```diff
@@ -313,14 +312,7 @@ public void run() {
 
       if (numSelectedTours < 2) {
 
-         final Runnable deleteRunnable = new Runnable() {
-            @Override
-            public void run() {
-
-               // delete selected tours
-               deleteTours(selectedTourIDs, treeSelection[0], selectionForDeletedTours, null);
-            }
-         };
+         final Runnable deleteRunnable = () -> deleteTours(selectedTourIDs, treeSelection[0], selectionForDeletedTours, null);
          BusyIndicator.showWhile(Display.getCurrent(), deleteRunnable);
 
       } else {
```

Comment:
The following lines could also be converted into a lambda expression, it's almost the same code